scene/3d/reflection_probe.cpp

```diff
@@ -85,38 +85,38 @@ float ReflectionProbe::get_lod_threshold() const {
 	return lod_threshold;
 }
 
-void ReflectionProbe::set_extents(const Vector3 &p_extents) {
-	extents = p_extents;
+void ReflectionProbe::set_size(const Vector3 &p_size) {
+	size = p_size;
 
 	for (int i = 0; i < 3; i++) {
-		if (extents[i] < 0.01) {
-			extents[i] = 0.01;
+		if (size[i] < 0.005) {
+			size[i] = 0.005;
```

Comment:
I have a feeling changing this threshold value to be smaller will cause issues below. As you can see the rest of the code in this function assumes that the size is at least 0.01

Reply:
In the version I'll push soon (once I've done some verification) I've kept this minimum size to 0.01 as I remembered several parts of the editor use and display 0.01 as the smallest value. Instead the lines below now divide the whole expression by 2 which I think would be the intended behaviour.

Reply:
[I posted this a bit earlier but it disappeared when I deleted a duplicate of it that got shown]
In the version I'll push soon once I've verified a few things, I've kept this minimum size bound to 0.01 because I remembered several parts of the editor use this value as the lowest setting. Instead the lines below now divide the whole expression by 2 which seems correct to me.